html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>
+      <label for="email">Email</label>
+      <br>
+      <input type="text" id="email">
+    </div>
+    <div>
+      <label for="coding">Why is coding cool?</label>
+    </div>
+    <select id="coding">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" id="html" name="answer" value="HTML">
+    <label for="html">HTML</label>
+    <input type="radio" id="css" name="answer" value="CSS">
+    <label for="CSS">CSS</label>
+    <input type="radio" id="javascript" name="answer" value="JavaScript">
+    <label for="JavaScript">JavaScript</label>
```

Comment:
Same deal here, the casing is different between the `id` and `for` values

Reply:
<img width="394" alt="Screen Shot 2022-05-10 at 1 13 17 PM" src="https://user-images.githubusercontent.com/103629526/167729322-60cd81b2-c02d-47e1-a7a0-9fa60591582f.png">